packages/fuselage/src/components/Sidebar/Sidebar.styles.scss

```diff
@@ -99,7 +99,7 @@ $sidebar-banner-background-default: theme(
 );
 $sidebar-banner-color-default: theme(
   'sidebar-banner-color-default',
-  colors.font(default)
+  colors.font(pure-white)
```

Comment:
O dont think this is the right way ,  please @juliajforesti @dougfabris  could you check plz?